content/pt-br/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md

```diff
@@ -0,0 +1,291 @@
+
+## Fazendo com que o iptables enxergue o tráfego agregado
+
+Se assegure de que o módulo `br_netfilter` está carregado. Isso pode ser feito executando o comando `lsmod | grep br_netfilter`. Para carrega-lo explicitamente execute `sudo modprobe br_netfilter`.
+
+Você também deve se assegurar de que a configuração `net.bridge.bridge-nf-call-iptables` no seu `sysctl` está com o valor 1. Um requerimento para que o iptables de cada um dos seus nós Linux enxergue o tráfego agregado corretamente. Como no exemplo abaixo:
+
+```bash
+cat <<EOF | sudo tee /etc/modules-load.d/k8s.conf
+br_netfilter
+EOF
+
+cat <<EOF | sudo tee /etc/sysctl.d/k8s.conf
+net.bridge.bridge-nf-call-ip6tables = 1
+net.bridge.bridge-nf-call-iptables = 1
+EOF
+sudo sysctl --system
+```
+
+Para mais detalhes veja a página [Requisitos do plugin de rede](/pt-br/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins/#network-plugin-requirements).
+
+## Verificando as portas necessárias
+
+### Nós da camada de gerenciamento
+
+| Protocolo | Direção | Intervalo de portas | Propósito       | Utilizada por             |
+|----------|-----------|------------|-------------------------|---------------------------|
+| TCP      | Entrada   | 6443\*     | API do Kubernetes       | Todos                      |
+| TCP      | Entrada   | 2379-2380  | API servidor cliente do etcd | kube-apiserver, etcd      |
+| TCP      | Entrada   | 10250      | API do kubelet          | kubeadm, Camada de gerenciamento       |
+| TCP      | Entrada   | 10251      | kube-scheduler          | kubeadm                      |
+| TCP      | Entrada   | 10252      | kube-controller-manager | kubeadm                      |
+
+### Nós de processamento
+
+| Protocolo | Direção | Intervalo de portas | Propósito       | Utilizada por             |
+|----------|-----------|-------------|-----------------------|-------------------------|
+| TCP      | Entrada   | 10250       | API do kubelet        | kubeadm, Camada de gerenciamento     |
+| TCP      | Entrada   | 30000-32767 | Serviços NodePort†    | Todos                     |
+
+† Intervalo padrão de portas para o [Serviços NodePort](/docs/concepts/services-networking/service/).
+
+Quaisquer portas marcadas com * podem ser substituídas. Dito isso, tenha certeza de que qualquer porta que você tenha customizado esteja aberta.
+
+Embora portas etcd sejam inclusas nos nós da camada de gerenciamento, você também pode hospedar seu próprio cluster etcd externamente, ou em portas customizadas. 
+
+O plugin de rede dos pods que você utiliza também pode requer que algumas portas estejam abertas. Dito que essas portas podem diferir dependendo do plugin, por favor leia a documentação dos plugins sobre quais portas serão necessárias abrir.
+
+## Instalando agente de execução {#instalando-agente-de-execucao}
```

Comment:
This is a bit strange. What if: "Instalando o agente de execução de contêineres" ?

Reply:
Better indeed.